core/domain/protofiles/text_classifier.proto

```diff
@@ -0,0 +1,21 @@
+// coding: utf-8
+//
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+message TextClassifier {
+  string model_json = 1;
```

Comment:
Just one check -- I remember getting the impression that we were not actually planning to use json to represent the models -- there was some other type of data transfer format that was better (in fact I thought that's why we're using protos in the first place). Did I remember wrong? It seems a bit weird to me to be storing json inside a proto rather than the fields directly (but I don't have enough direct experience to feel super strongly about this).

Reply:
Yes, we are using proto so that we can optimize space required for models. However we will start using that when I upgrade TextClassifier in the next phase. There is no point of creating a protobuf based model for this TextClassifier as it will be deprecated and JSON can be used as a legacy way to keep ML pipeline intact.

Reply:
Could we use the final proto structure and convert the JSON to that? The persistence & communication of the data would be purely protobuf in that case, and only the uses of the proto would go to/from JSON. Then, the migration would just involve switching one path to the other rather than needing to perform a data migration.

Reply:
Yes, that will be done in phase 2 when the TextClassifier will be upgraded. Right now, laying out detailed structure is unnecessary as this structure is going to change in phase 2. Hence, I am using JSON so that we can at least test the ML pipeline which will be finished in this phase of milestones. Exact proto structure isn't really necessary for ML pipeline to work correctly.